home-tasks/src/test/java/com/epam/tc/hw2/utils/RegexChecker.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.tc.hw2.utils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RegexChecker {
+
+    public static String cutTimeOfLogString(String logStringWithTime) {
+        Matcher matcher = Pattern.compile("[a-zA-Z]+:.*").matcher(logStringWithTime);
+        StringBuilder stringBuilder = new StringBuilder();
+        while (matcher.find()) {
+            stringBuilder.append(matcher.group());
+        }
+        return stringBuilder.toString();
```

Comment:
Lets simplify this code
```suggestion
        return logStringWithTime.replaceFirst("^[\\d]{2}:[\\d]{2}:[\\d]{2} ", "");
```